src/librustc_data_structures/tiny_list/tests.rs

```diff
@@ -111,6 +111,27 @@ fn bench_insert_one(b: &mut Bencher) {
     })
 }
 
+#[bench]
+fn bench_contains_empty(b: &mut Bencher) {
+    b.iter(|| {
+        TinyList::new().contains(&1)
```

Comment:
You need to `black_box` the list, here and in the other tests, so the benchmark is actually doing anything.

Reply:
Added.